app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -391,6 +403,21 @@ def parse_replication_controllers(container_replicator)
       new_result
     end
 
+    def parse_component_status(container_component_status)
+      new_result = {}
+      component_condition = container_component_status.conditions.first
```

Comment:
@dkorn remember me, why do we take only the first one? Is there any documentation that states that there's only one condition for components?